ukvm/ukvm-net.c

```diff
@@ -177,10 +179,16 @@ static int handle_exit(struct kvm_run *run, int vcpufd, uint8_t *mem)
 
 static int handle_cmdarg(char *cmdarg)
 {
-    if (strncmp("--net=", cmdarg, 6))
+    if (!strncmp("--net=", cmdarg, 6)) {
+        netiface = cmdarg + 6;
+        return 0;
+    } else if (!strncmp("--mac=", cmdarg, 6)) {
+        snprintf(netinfo.mac_str, sizeof(netinfo.mac_str), cmdarg + 6);
```

Comment:
This should pass `%s` as the format string to `snprintf()` to prevent format-string-injection bugs or use `strncpy()` / `strlcpy()`.